<launch>
  <node pkg="robotino_local_move" type="robotino_local_move_server_node" name="robotino_local_move_server_node" output="screen">
    <param name="min_forward_vel" value="0.05" />
    <param name="max_forward_vel" value="0.1" />
    <param name="min_forward_vel_distance" value="0.1" />
    <param name="max_forward_vel_distance" value="0.2" />
    <param name="min_rotational_vel" value="0.1" />
    <param name="max_rotational_vel" value="0.3" />
    <param name="min_rotational_vel_distance" value="0.05" />
    <param name="max_rotational_vel_distance" value="0.2" />
  </node>
</launch>